fix emalil formatting (#21)
nhs-r-community/conference-checkin: backend/email_body.md

The nearest underground stations are:

- Euston Square
- Euston
- Warren Street 

> - King’s Cross St Pancras

Reviewer: ```suggestion
- King's Cross St Pancras
```

might be better to use '

The nearest mainline stations are:

- Euston

> - King’s Cross

Reviewer: ```suggestion
- King's Cross
```

If you’re using a taxi, there are drop off points by our:

- main visitor entrance at 215 Euston Road (raised kerb)

> - staff-only entrance on Gower Place (raised and dropped kerb). It’s a short distance from here to our main visitor entrance at the front of the building.

Reviewer: ```suggestion
- staff-only entrance on Gower Place (raised and dropped kerb). It's a short distance from here to our main visitor entrance at the front of the building.
```
